### PR TITLE
set jenkins-lts as the recommended way of installing Jenkins

### PR DESCRIPTION
**Pull request by art-divin**
changed from jenkins to jenkins-lts as a recommended way of installing Jenkins

### CODE REVIEW

**Comment by art-divin** [comment]
@googlebot I signed it!

**Review by lyndsey-ferguson — APPROVED** [review]
https://www.jenkins.io/download/lts/macos/

**Review by rogerluan — COMMENTED** [review]
This is cool :) thanks for the docs @lyndsey-ferguson !

Thanks for your first contribution @art-divin ❤️

File: docs/best-practices/continuous-integration/jenkins.md [hunk 1]
```diff
@@ -5,13 +5,13 @@
 The recommended way to install [Jenkins](http://jenkins-ci.org/) is through [homebrew](http://brew.sh/):
```
Comment (rogerluan) [inline docs/best-practices/continuous-integration/jenkins.md:5]:
```suggestion
The recommended way to [install Jenkins on macOS](https://www.jenkins.io/download/lts/macos/) is through [homebrew](http://brew.sh/):
```

**Review by rogerluan — CHANGES_REQUESTED** [review]

**Review by rogerluan — APPROVED** [review]
Thanks again for this contribution!

**Pull request merged**